ansible_collections/arista/avd/tests/unit/filters/test_range_expand.py

```diff
@@ -25,6 +25,22 @@
     "65100.0-4",
     "65100.0-2,65200.1-2",
     "1-2.0-1",
+    "Ethernet(1)",
+    "Ethernet(1,2,3)",
+    "Ethernet(1)/(4,5)",
+    "(1,2)",
+    "1,(2,3)/5",
+    "Eth(10-14)",
+    "Eth(101-103)/(5,7)",
+    "Eth(1,2",
```

Comment:
Yes, this is invalid syntax for parentheses "()" or range "-".
As discussed with @ClausHolbechArista , our requirement was to only expand if it falls under "()" or "-" otherwise let it pass through, since we want to add more expand parameters later.
So in this case since the syntax does not matches with "()" or "-" it is going to pass through with the output of prefix + split(",")
which would be "Eth(1,2" --> ["Eth(1", "Eth2"]

I understand this would not be the expected output from the function we could discuss about errors over call if you like :)